lib/src/http/types.ts

```diff
@@ -7,18 +7,21 @@ export interface PullFilters {
     excludeNestedFolders?: boolean;
   }[];
   variants?: { id: string }[];
+  statuses?: ITextStatus[];
 }
 
 export interface PullQueryParams {
   filter: string; // Stringified PullFilters
   richText?: "html";
 }
+export const ZTextStatus = z.enum(["NONE", "WIP", "REVIEW", "FINAL"]);
+export type ITextStatus = z.infer<typeof ZTextStatus>;
 
 const ZBaseTextEntity = z.object({
   id: z.string(),
   text: z.string(),
   richText: z.string().optional(),
-  status: z.string(),
+  status: ZTextStatus,
```

Comment:
weird this was string originally!

Reply:
Actually it might be better to keep it as `string` for now. It makes changes to the status schema easier (does not require an update to the CLI to fetch new types of statuses) Especially if the status is essentially "unused" from a typing perspective on output